packages/apps/plugins/plugin-native/src/NativePlugin.tsx

```diff
@@ -48,6 +48,7 @@ const initializeNativeApp = async (plugins: Plugin[]) => {
   //
 
   let itemsMac = '';
+  // TODO(mjamesderocher) This isn't working
```

Comment:
When this came up before, we moved on from it, since we found a different solution. But I think we are creating our own `process` object? https://discord.com/channels/837138313172353095/1180035274345025617/1186723113044557844

If that's the case, is it possible we are overwriting what SSC is providing?

Reply:
Yeah, we do have a `process` polyfill. That sounds likely what's happening.

Reply:
@mjamesderocher here's an issue I just made for this https://github.com/dxos/dxos/issues/5689 perhaps you could link it in the TODO

Reply:
Ok. Since at first, we are only planning on releasing a desktop app for Mac OS, I removed the conditional here so that those menu items will show up.